### PR TITLE
Exam

### PR DESCRIPTION
**Pull request by chebelok**

### CODE REVIEW

File: Tasks/replace.js [hunk 1]
```diff
@@ -6,18 +6,19 @@ const Replace = (str, substr, newstr) => {
...
+        res += start + newstr + str;
+        return res;
       }
-    } while (true);
+    }
   }
 };
 
-const result = Replace('Hello <username> and bye!', '<username>', 'Marcus');
+const result = Replace('Hello <username> and bye!', '<username>', 'kekw');
```
Comment (chebelok) [inline Tasks/replace.js:23]: ooops, "kekw" is a little mistake, forget to save correct name, before pushing on githhub, sorry :)

**Comment by tshemsedinov** [comment]
Improve naming: `i` (if for in), `obj3`

**Comment by tshemsedinov** [comment]
- Respect `camelCase`
- Use eslint and prettier

**Comment by tshemsedinov** [comment]
Use return-early strategy

**Pull request closed without merging**